.github/workflows/atlantis-image.yml

```diff
@@ -49,13 +49,13 @@ jobs:
     - name: populate release version
       if: ${{ github.event_name == 'release'}}
       run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
-    - name: Build and push atlantis:$RELEASE_VERSION image
+    - name: Build and push atlantis:${{ env.RELEASE_VERSION }} image
```

Comment:
Interesting, I think this used to working fine at some point before. 

But now when I looked back at 0.17.3 release build (https://github.com/runatlantis/atlantis/runs/3478283112), the RELEASE_VERSION shows unset in the log.

I dont think this is quite a right fix. You can see my test build in here, https://github.com/chenrui333/github-action-test/runs/4715966380

The image does get successfully tagged.
![image](https://user-images.githubusercontent.com/1580956/148236282-894bcc07-8d6a-4fff-8eae-2202073a12b9.png)

Reply:
This is the workflow ref, https://github.com/chenrui333/github-action-test/blob/main/.github/workflows/release.yml

Reply:
I don't quite follow you. The first release that happened since the changes from #1943 got merged was the version 0.17.6 release. You can see the job from that run here: https://github.com/runatlantis/atlantis/runs/4564698591?check_suite_focus=true.

The workflow you are linking to is running the `docker build` command directly, which means environment variables are expanded. This was changed in [.github/workflows/atlantis-image.yml](https://github.com/runatlantis/atlantis/blob/master/.github/workflows/atlantis-image.yml) in #1943 to instead use the `docker/build-push-action` action, which doesn't expand environment variables, which is why the environment variable instead has to be accessed as via the template language.

Does that make sense?

Reply:
Aha, that makes total sense.

Reply:
Thanks for the fix.

Reply:
You're welcome! I'm looking forward to running Atlantis on ARM :)

Reply:
Just a note, the built images for ARM don't work properly (at least testing on Amazon Fargate / ARM).  Keep getting the following error:

`standard_init_linux.go:228: exec user process caused: exec format error`

Reply:
@virgofx Did you use this image `ghcr.io/runatlantis/atlantis:latest@sha256:66d431df3cd1893175abf7133aeef96f9ab212a3c260daf8ef2a54042259f503`? Mind creating a bug ticket?

Reply:
Yep confirmed using 
`ghcr.io/runatlantis/atlantis:latest@sha256:66d431df3cd1893175abf7133aeef96f9ab212a3c260daf8ef2a54042259f503`
results in:
`standard_init_linux.go:228: exec user process caused: exec format error`

I'll create a new issue.

Reply:
Confirmed. Spun out a new issue: https://github.com/runatlantis/atlantis/issues/2001

Reply:
Thanks @virgofx!